.github/workflows/maven.yml

```diff
@@ -5,9 +5,9 @@ name: Build
 
 on:
   push:
-    branches: [ master, development ]
+    branches: [ development, development_rework ]
   pull_request:
-    branches: [ master, development ]
+    branches: [ development, development_rework ]
```

Comment:
This looks outdated.

Reply:
Yes, this and the prototype code will have to go eventually.